corehq/apps/builds/models.py

```diff
@@ -132,6 +133,17 @@ def get_build(cls, version, build_number=None, latest=False):
     def all_builds(cls):
         return cls.view('builds/all', include_docs=True, reduce=False)
 
+    @classmethod
+    @quickcache([], timeout=5 * 60) #This seems to be not working.
```

Comment:
E261 at least two spaces before inline comment
E262 inline comment should start with '# '